.azdo/ci.yml

```diff
@@ -3,15 +3,14 @@
 # vmImages and goVersions specified by the parameters.
 
 parameters:
-  goVersions: [ ]
-  vmImages: [ ]
+  goVersions: []
```

Comment:
eh... just noticed that my yaml formatter fired here -- let me know if I should revert.